pkgs/by-name/ed/edmarketconnector/package.nix

```diff
@@ -0,0 +1,81 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  python3Full,
+  makeWrapper,
+  makeDesktopItem,
+}:
+let
+  pkgVersion = "5.12.2";
```

Comment:
Why name it `pkgVersion` instead of `version`? With just `version` its also possible to inherit instead of insert for the derivation version.